admin/class-metabox.php

```diff
@@ -1210,6 +1218,33 @@ function column_sort_orderby( $vars ) {
 	}
 
 	/**
+	 * Hide certain columns if the user hasn't chosen which columns to hide
```

Comment:
I assume the columns you wish to hide, are the title, metadesc and focuskeyword (taken from code). Please also elaborate this in the phpdoc